packages/upgrade-tests/src/tests/automations.upgrade.test.ts

```diff
@@ -0,0 +1,21 @@
+import { BudibaseClient } from "../index"
+import { it } from "../utils/upgradeTest"
+
+describe("Automation Upgrade Tests", () => {
+  let client: BudibaseClient
+
+  beforeAll(async () => {
+    client = await BudibaseClient.authenticated()
+  })
+
+  it("should preserve automation counts", {
+    pre: async () => {
+      const automations = await client.automation.fetch()
+      return { automationCount: automations.length }
+    },
+    post: async ({ automationCount }) => {
```

Comment:
I like this idea of the pre output being the post input :)

Reply:
It's clean, isn't it? Allows you a lot of freedom in how you want to verify the upgrade was successful. Also wonderfully typesafe 😁